_includes/footer.html

```
@@ -1,27 +1,27 @@
+				<img src="{{ "/assets/svg/discord.svg" | relative_url }}" width="24" height="24" alt="" aria-hidden="true" class="li img-invert">
+			</a>
+			<a href="mailto:contact@indierobloxwikis.org" aria-label="Email contact for IRWA">
+				<img src="{{ "/assets/icons/mail.svg" | relative_url }}" width="24" height="24" alt="" aria-hidden="true" class="li img-invert">
+			</a>
+		</div>
+	</div>
+	<div class="footer__right">
+		<ul class="footer__nav">
+			<li><a href="{{ "/about" | relative_url }}" class="invert">About</a></li>
+			<li><a href="{{ "/blog" | relative_url }}" class="invert">Blog</a></li>
+			<li><a href="{{ "/policies" | relative_url }}" class="invert">Policy</a></li>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)